### PR TITLE
Update static instances list

### PR DESCRIPTION
**Pull request by SamantazFox**
- Remove dead instances
- Add new ones
- Add country flag icon

### CODE REVIEW

File: src/index.pug [hunk 1]
```diff
@@ -50,20 +50,24 @@ html(lang="en")
...
+							{url: "iv.ggtyler.dev", flag: "🇨🇦"},
+							{url: "invidious.0011.lt", flag: "🇱🇹"},
+							{url: "inv.zzls.xyz", flag: "🇨🇱"},
+							{url: "invidious.protokolla.fi", flag: "🇫🇮"},
 						]
 					for instance in instances
-						li: a(href=`https://${instance}`)= instance
+						li
+							a(href=`https://${instance["url"]}`)= instance["url"]
+							span= '(' + instance["flag"] + ')'
```
Comment (cloudrac3r) [inline src/index.pug:71]: I would do it this way:

```
					for instance in instances
						li
							= `${instance.flag} `
							a(href=`https://${instance.url}`)= instance.url
```

**Review by cloudrac3r — CHANGES_REQUESTED** [review]
Overall, nice job!

Since the API provides the flags and region info, I would also change the javascript-on version to include it too. You can add this to main.js line 68:

```
					createElement("td", {textContent: `${entry.details.flag} ${entry.details.region}`}),
```

Since the table now has a new column, you'll also need to adjust index.pug lines 40-46 to add the column header:

```
						th(scope="col") Region
```

and increase the span to 4:

```
						td(colspan="4" data-loading-message="Loading instances...").loading-td Trying to run scripts...
```

**Comment by SamantazFox** [comment]
Thanks a lot for your review! I made the proposed changes.

File: src/index.pug [hunk 2]
```diff
@@ -37,33 +37,38 @@ html(lang="en")
...
+							{url: "vid.priv.au", flag: "🇸🇬"},
+							{url: "iv.ggtyler.dev", flag: "🇨🇦"},
+							{url: "invidious.0011.lt", flag: "🇱🇹"},
+							{url: "inv.zzls.xyz", flag: "🇨🇱"},
+							{url: "invidious.protokolla.fi", flag: "🇫🇮"},
 						]
 					for instance in instances
-						li: a(href=`https://${instance}`)= instance
+						li
+							= `instance.flag `
```
Comment (cloudrac3r) [inline src/index.pug:71]: I just tested this locally. Please change this to ``= `${instance.flag} ` `` (adding javascript string interpolation) and then we should be good to go!

**Review by cloudrac3r — CHANGES_REQUESTED** [review]

**Review by cloudrac3r — APPROVED** [review]

**Comment by SamantazFox** [comment]
@cloudrac3r thanks for the feedback :)

**Pull request merged**